frontend/components/QuestionPages/QuestionAnswers.tsx

```diff
@@ -1,48 +1,115 @@
+import { useState } from 'react';
 import MatchMediaWrapper from '@/components/MatchMediaWrapper';
+import Image from 'next/image';
+import X from 'public/remove 1.png';
```

Comment:
Nitpicky, but maybe standardize the asset names somehow -- I see a mix of spaces, hyphens, and uppercase in these imports.